content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -9,197 +9,81 @@ card:
 
 {{% capture overview %}}
 
-This page shows you how to install [Minikube](/docs/tutorials/hello-minikube), a tool that runs a single-node Kubernetes cluster in a virtual machine on your personal computer.
+This page shows you how to install [minikube](https://minikube.sigs.k8s.io/), a tool that runs a single-node Kubernetes cluster on your personal computer.
 
 {{% /capture %}}
 
 {{% capture prerequisites %}}
 
-{{< tabs name="minikube_before_you_begin" >}}
-{{% tab name="Linux" %}}
-To check if virtualization is supported on Linux, run the following command and verify that the output is non-empty:
-```
-grep -E --color 'vmx|svm' /proc/cpuinfo
-```
-{{% /tab %}}
+To run minikube, you'll need:
 
-{{% tab name="macOS" %}}
-To check if virtualization is supported on macOS, run the following command on your terminal.
-```
-sysctl -a | grep -E --color 'machdep.cpu.features|VMX'
-```
-If you see `VMX` in the output (should be colored), the VT-x feature is enabled in your machine.
-{{% /tab %}}
-
-{{% tab name="Windows" %}}
-To check if virtualization is supported on Windows 8 and above, run the following command on your Windows terminal or command prompt.
-```
-systeminfo
-```
-If you see the following output, virtualization is supported on Windows.
-```
-Hyper-V Requirements:     VM Monitor Mode Extensions: Yes
-                          Virtualization Enabled In Firmware: Yes
-                          Second Level Address Translation: Yes
-                          Data Execution Prevention Available: Yes
-```
-
-If you see the following output, your system already has a Hypervisor installed and you can skip the next step.
-```
-Hyper-V Requirements:     A hypervisor has been detected. Features required for Hyper-V will not be displayed.
-```
-
-
-{{% /tab %}}
-{{< /tabs >}}
+* 2GB of free memory
+* 20GB of free disk space
+* Internet connection
+* A compatible container or virtual machine manager, such as: [Docker](https://minikube.sigs.k8s.io/docs/docs/drivers/docker), [Hyperkit](https://minikube.sigs.k8s.io/docs/docs/drivers/hyperkit), [Hyper-V](https://minikube.sigs.k8s.io/docs/docs/drivers/hyperv), [KVM](https://minikube.sigs.k8s.io/docs/docs/drivers/kvm2), [Parallels](https://minikube.sigs.k8s.io/docs/docs/drivers/parallels), [Podman](https://minikube.sigs.k8s.io/docs/docs/drivers/podman), [VirtualBox](https://minikube.sigs.k8s.io/docs/docs/drivers/virtualbox), or [VMWare](https://minikube.sigs.k8s.io/docs/docs/drivers/vmware)
```

Comment:
nit: This looks like it would work better as a Markdown list. Also, I'm not clear if that's intended to seem like an exhaustive list or not.

Reply:
It isn't entirely exhaustive, but it does include the most inquired about options. I'm OK to narrow the list down, but I felt like comma-separated made it feel more conversational.

Reply:
It's fine to leave this as you've suggested @tstromberg - I can't propose any better wording.